### ISSUE TITLE
angular-checklist cli

### ISSUE THREAD

**Issue by nongrata081** [typebytes/angular-checklist#52]
Would be nice to have an angular-checklist cli, that would be able to run against an angular project and generate a report, same way as [lighthouse](https://github.com/GoogleChrome/lighthouse) does.

**Comment by d3lm** [comment]
Yep agreed. But I think this will be rather difficult to achieve because how do you figure out if an item is implemented? I mean every project is different and developers might implement these items in different ways. Finding something that works reliably across many projects is very tricky if not impossible. But if you have an idea, I am more than happy to discuss this because I really like the idea of having a CLI that creates an automatic report.

@KwintenP Any ideas or thoughts on this one?
